tests/Integration/Core/Form/IdentifiableObject/DataHandler/ProductFormDataHandlerChecker.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Tests\Integration\Core\Form\IdentifiableObject\DataHandler;
+
+use PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataHandler\FormDataHandlerInterface;
+
+class ProductFormDataHandlerChecker implements FormDataHandlerInterface
+{
+    /**
+     * @var FormDataHandlerInterface
+     */
+    private $productFormDataHandler;
+
+    /**
+     * @var ?array
```

Comment:
First time I see @var ?array in PS code.
We used to have 'array|null'
Just noticing